demo/src/main/java/com/novoda/merlin/demo/presentation/RxJava2DemoActivity.java

```diff
@@ -27,8 +27,8 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.main);
         viewToAttachDisplayerTo = findViewById(R.id.displayerAttachableView);
-        networkStatusDisplayer = new NetworkStatusDisplayer(getResources(), merlinsBeard);
         merlinsBeard = MerlinsBeard.from(this);
+        networkStatusDisplayer = new NetworkStatusDisplayer(getResources(), merlinsBeard);
```

Comment:
Nullability annotations do highlight misuses in most scenarios. Unfortunately there's no way to say "everything is non-null by default in this codebase, unless annotated as `@Nullable`". At most you can do it per-package. Or, use Kotlin :D That would actually enforce it and annotate for Java too.